telegram/ext/_handlers/conversationhandler.py

```diff
@@ -111,12 +120,75 @@ def resolve(self) -> object:
 
         res = self.task.result()
         if res is None and self.old_state is None:
+            branch_coverage["103"] = True
             res = ConversationHandler.END
         elif res is None:
+            branch_coverage["104"] = True
             # returning None from a callback means that we want to stay in the old state
             return self.old_state
 
         return res
+    
+    """ def print_coverage():
+        for branch, hit in branch_coverage.items():
+            print(f"{branch} was {'hit' if hit else 'not hit'}") """
+
```

Comment:
Probably want to remove this?

Reply:
It is needed for the assignment presentation.

Reply:
why is it commented?